demos/text-field.html

```diff
@@ -382,6 +392,22 @@ <h2>CSS Only Text Field box</h2>
                    placeholder="Name">
           </div>
         </div>
+        <div>
+          <input type="checkbox" class="demo-disabled" id="css-only-box-disable">
```

Comment:
Nit: This page currently has a mix of "disable" and "disabled" for checkbox IDs... can we make them consistent?